src/gtk/toplevel.cpp

```diff
@@ -675,6 +690,13 @@ bool wxTopLevelWindowGTK::Create( wxWindow *parent,
     gdk_set_program_class(wxTheApp->GetAppDisplayName().utf8_str());
 #endif
 
+    // This value may be incorrect as here it's just set to the scale factor of
+    // the primary monitor, but it's better than setting it to 1, as chances
+    // are that the window will be created on the primary monitor or, maybe,
+    // the other monitors use the same scale factor anyhow. And if this isn't
+    // the case, it will corrected when we get "configure-event" from GTK.
+    m_scaleFactor = GetContentScaleFactor();
```

Comment:
This is too early, `m_widget` needs to be valid.

Reply:
Ugh, yes, of course, thanks again.